src/main/java/com/gizasystems/automationexercise/pages/LogoutUser.java

```diff
@@ -0,0 +1,29 @@
+package com.gizasystems.automationexercise.pages;
+
+import com.shaft.driver.SHAFT;
+import io.qameta.allure.Step;
+import org.openqa.selenium.By;
+
+public class LogoutUser {
+
+    //Variables
+    private SHAFT.GUI.WebDriver driver;
+    private final String url = System.getProperty("baseUrl");
```

Comment:
Where is the link to the page itself here?

Reply:
It is not a page 
it is just a button in the navigation bar